src/psyclone/dynamo0p3.py

```diff
@@ -6528,14 +6519,22 @@ def _compute_from_field(self, field):
 
 
 class DynLoop(Loop):
-    ''' The Dynamo specific Loop class. This passes the Dynamo
+    '''
+    The Dynamo specific Loop class. This passes the Dynamo
```

Comment:
I would suggest using "LFRic" in this docstring. Also, "LFRic-specific Loop class" (compound adjective).

Reply:
Oh yes, done.